src/Services/OrderBookReader.cs

```diff
@@ -17,18 +17,16 @@ public class OrderBookReader : IOrderBookReader
 
         public OrderBookReader(RabbitMqSubscriptionSettings settings,
             IOrderBooksHandler orderBooksHandler,
-            ILog log)
+            ILogFactory logFactory)
         {
             _orderBooksHandler = orderBooksHandler;
 
-            _subscriber =
-                new RabbitMqSubscriber<OrderBook>(settings,
-                        new ResilientErrorHandlingStrategy(log, settings, TimeSpan.FromSeconds(10),
-                            next: new DefaultErrorHandlingStrategy(log, settings)))
+            var strategy = new ResilientErrorHandlingStrategy(logFactory, settings, TimeSpan.FromSeconds(10),
+                next: new DefaultErrorHandlingStrategy(logFactory, settings));
+            _subscriber = new RabbitMqSubscriber<OrderBook>(logFactory, settings, strategy)
                     .SetMessageDeserializer(new JsonMessageDeserializer<OrderBook>())
                     .SetMessageReadStrategy(new MessageReadWithTemporaryQueueStrategy())
-                    .Subscribe(HandleData)
-                    .SetLogger(log);
+                    .Subscribe(HandleData);
```

Comment:
It's better to limit prefetch count by using SetPrefetchLimit here

Reply:
Ok. Fixed.